docs/docs/index.md

```diff
@@ -35,33 +35,49 @@ Marathon packages are available from Mesosphere's [repositories](http://mesosphe
 Download and unpack the latest Marathon release.
 
 ``` bash
-$ curl -O http://downloads.mesosphere.com/marathon/v0.15.2/marathon-0.15.2.tgz
-$ tar xzf marathon-0.15.2.tgz
+$ curl -O http://downloads.mesosphere.com/marathon/v0.15.1/marathon-0.15.1.tgz
+$ tar xzf marathon-0.15.1.tgz
 ```
 
 SHA-256 checksums are available by appending `.sha256` to the URLs.
 
+### Running in Production Mode
+
+To launch Marathon in *production mode*, you need to have both
+ZooKeeper and Mesos running. The following command launches
+Marathon in *production mode*. Point your web browser to
+`localhost:8080` and you should see the [Marathon UI]({{ site.baseurl }}/docs/marathon-ui.html).
+
+``` console
+$ ./bin/start --master zk://zk1.foo.bar:2181,zk2.foo.bar:2181/mesos --zk zk://zk1.foo.bar:2181,zk2.foo.bar:2181/marathon
+```
+
+Marathon uses `--master` to find the Mesos masters, and `--zk` to find ZooKeepers
+for storing state. They are separate options because Mesos masters can be
+discovered in other ways as well.
+
+For all configuration options, see the [command line flags](command-line-flags.html) doc. For more information on the high-availability feature of Marathon, see the [high availability](high-availability.html) doc.
+
 ### Versioning
 
-Starting with version 0.9.0 Marathon will adhere to [semantic versioning](http://semver.org).
-That means we are committed to keep our documented REST API compatible across releases unless we change the MAJOR version
-(the first number in the version tuple). If you depend on undocumented features, please tell us about them by [raising a GitHub issue](https://github.com/mesosphere/marathon/issues/new). API parts which we explicitly marked as EXPERIMENTAL are exempted from this rule. We will not introduce new features in PATCH version increments (the last number in the version tuple).
+As of version 0.9.0, Marathon adheres to [semantic versioning](http://semver.org).
+That means our documented REST API remains compatible across releases unless we change the MAJOR version
+(the first number in the version tuple). If you depend on undocumented features, please tell us about them by [raising a GitHub issue](https://github.com/mesosphere/marathon/issues/new). Portions of the API marked as EXPERIMENTAL are exempt from this rule. We do not introduce new features in PATCH version increments (the last number in the version tuple).
 
-We might change the command line interfaces of the Marathon server process in rare cases in a MINOR version upgrade.
-Please check the release notes for these.
+In rare cases, we may change the command line interfaces of the Marathon server process in a MINOR version upgrade. The release notes document these instances.
```

Comment:
Is there another way to say "the command line interfaces of the Marathon server process"? Or can someone explain to me what this means?